docs/admin_api/rooms.md

```diff
@@ -264,3 +264,34 @@ Response:
 
 Once the `next_token` parameter is no longer present, we know we've reached the
 end of the list.
+
+# DRAFT: Room Details API
+
+The Room Details admin API allows server admins to get all details of a room,
+including a list of all room members.
+
+This API is still a draft and details might change!
+
+## Usage
+
+A standard request:
+
+```
+GET /_synapse/admin/v1/rooms/<room_id>
+
+{}
+```
+
+Response:
```

Comment:
It would be nice to expand upon what fields are returned. Maybe we can refer to the documentation under "List Room API" for an explanation of the fields if they're identical?